common/source/docs/common-msp-osd-overview-4.2.rst

```diff
@@ -43,64 +43,63 @@ OSD Panel Items
 +---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| OSDn_GPSLAT   |                                                                                                                                                                                                                                                                                                      |
+| OSDn_GPSLAT   | GPS Latitude in decimal format                                                                                                                                                                                                                                                                       |
 +---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| OSDn_GPSLONG  |                                                                                                                                                                                                                                                                                                      |
+| OSDn_GPSLONG  | GPS Longitude in decimal format                                                                                                                                                                                                                                                                      |
 +---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| OSDn_GSPEED   | This item shows ground speed unless ``OSDn_ASPEED_EN`` = 1 in which case it will show true airspeed if an airspeed sensor is present or estimated airspeed otherwise. The position on screen is set by ``OSDn_GSPEED_X`` and ``OSDn_GSPEED_Y`` regardless of the value of ``OSDn_ASPEED_EN``         |
+| OSDn_GSPEED   | This item shows ground speed unless OSDn_ASPEED_EN = 1 in which case it will show true airspeed if an airspeed sensor is present or estimated airspeed otherwise. The position on screen is set by OSDn_GSPEED_X and OSDn_GSPEED_Y irregardless of the value of OSDn_ASPEED_EN                       |
```

Comment:
my  practice is to always highlite a parameter name, if possible with it linked....when talking about a muli-instance one I use bold (double back-ticks)
also "irregardless" while defined in English, is a non-standard word...use "regardless" instead